Jenkinsfile.deploy

```diff
@@ -42,37 +56,118 @@ pipeline {
                     if (params.DISPLAY_NAME != '') {
                         currentBuild.displayName = params.DISPLAY_NAME
                     }
+
+                    if (isRelease()) {
+                        assert getProjectVersion() != ''
+                    }
                 }
             }
             post {
                 success {
                     script {
                         setDeployPropertyIfNeeded('git.branch', getBuildBranch())
                         setDeployPropertyIfNeeded('git.author', getGitAuthor())
+                        setDeployPropertyIfNeeded('project.version', getProjectVersion())
+                        setDeployPropertyIfNeeded('release', isRelease())
```

Comment:
maybe add `KOGITO_VERSION` as well in `deployment.properties`, as you will need it in promote pipeline

Reply:
I get the version there via a parameter, I suppose. But +1 to make it consistent.